package.json

```diff
@@ -9,8 +9,8 @@
     "test": "react-scripts test",
     "test:update": "npm test -- --watchAll=false --updateSnapshot",
     "test:ci": "npm test -- --watchAll=false --coverage --runInBand --no-cache",
-    "e2e:dev": "node -e 'require(\"./src/e2eConfig\").e2eRunDev(\"http://localhost:3000\")'",
-    "e2e": "PUBLIC_URL=http://localhost:3000 npm run build && node -e 'require(\"./src/e2eConfig\").e2eRun(\"3000\")'",
+    "e2e:dev": "cypress open",
```

Comment:
I think it would be nice to make **e2e:dev** and **e2e** run as atomic commands. Right now you first need to spin app with **npm start** to make it work. If you don't do that you are welcomed by warning (all test fail):
<img width="797" alt="Screenshot 2019-08-05 at 13 04 52" src="https://user-images.githubusercontent.com/20565536/62531278-6c8c5800-b842-11e9-939c-e3a629d2a12e.png">

Reply:
I've added small dev dependencies `concurrently` and `wait-on` that make it possible in this commit 👉https://github.com/BuildIt-Poland/daily-routine/pull/111/commits/064f50e16114818c29d6a9e745fb4373a3bcdc1c